### PR TITLE
Fixes of i18n on readme.md

### PR DESCRIPTION
**Pull request by GeorgeLimaDev**
#### What problem is this solving?

It relates to task [LOC-10873](https://vtex-dev.atlassian.net/browse/LOC-10873) and KR 23Q2-KR1 ([Review all developer portal documentation in English](https://docs.google.com/document/d/1VduAYpjqdazhyGH5DvD9c52pMu1Xfj4RDx48QNjTFU0/edit)).

[LOC-10873]: https://vtex-dev.atlassian.net/browse/LOC-10873?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -6,48 +6,48 @@
...
 
-:information_source: Currently, **the Product Price is the only app responsible for providing product price blocks for your theme**. Both [Product Summary Price](https://vtex.io/docs/components/all/vtex.product-summary/product-summary-price/) and [Product Price](https://vtex.io/docs/components/all/vtex.store-components/product-price/) blocks, respectively from [Product Summary](https://vtex.io/docs/components/all/vtex.product-summary/) and [Store Components](https://vtex.io/docs/components/all/vtex.store-components/) apps, were deprecated and therefore will no longer be evolved.
+> ℹ️ Currently, **Product Price is the only app that provides product price blocks for your theme**. Both, the [Product Summary Price](https://developers.vtex.com/docs/guides/vtex-product-summary-productsummaryprice) and [Product Price](https://developers.vtex.com/docs/guides/vtex-store-components-productprice/) blocks from the [Product Summary](https://developers.vtex.com/docs/guides/vtex-product-summary/) and [Store Components](https://developers.vtex.com/docs/guides/vtex-store-components/) apps respectively were deprecated and will no longer be developed.
```
Comment (julia-rabello) [inline docs/README.md:13]:
```suggestion
> ℹ️ Currently, **Product Price is the only app that provides product price blocks for your theme**. The [Product Summary Price](https://developers.vtex.com/docs/guides/vtex-product-summary-productsummaryprice) block from the [Product Summary](https://developers.vtex.com/docs/guides/vtex-product-summary/) app and the [Product Price](https://developers.vtex.com/docs/guides/vtex-store-components-productprice/) block from [Store Components](https://developers.vtex.com/docs/guides/vtex-store-components/) have been deprecated.
```

File: docs/README.md [hunk 2]
```diff
@@ -6,48 +6,48 @@
...
 
-## Configuration
+## Configuring Product Price
```
Comment (julia-rabello) [inline docs/README.md:15]:
```suggestion
## Configuration
```

File: docs/README.md [hunk 3]
```diff
@@ -6,48 +6,48 @@
...
 
-### Step 1 - Adding the Product Price app to your theme's dependencies
+### Step 1 - Adding the Product Price app to your theme dependencies
 
-In your theme's `manifest.json`, add the Product Price app as a dependency:
+In the theme `manifest.json`, add the Product Price app as a dependency:
```
Comment (julia-rabello) [inline docs/README.md:19]:
```suggestion
In the theme's `manifest.json` file, add the Product Price app as a dependency:
```

File: docs/README.md [hunk 4]
```diff
@@ -111,26 +111,25 @@ For example:
...
 
-And the block `product-installments` also has two additional props:
+| Prop name            | Type       | Description                                                                                                                                                                                                                                                                                                           | Default value |
+| -------------------- | ---------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `paymentSystemName`  | `string`   | This prop allows you to filter the listed installment options by a particular payment system. If not passed, the installments of the payment system with the most installment options will be rendered.                                                                                                               | `undefined`   |
```
Comment (julia-rabello) [inline docs/README.md:118]:
```suggestion
| `paymentSystemName`  | `string`   | Allows you to filter the listed installment options by a particular payment system. If not passed, the installments of the payment system with the most installment options will be rendered.                                                                                                               | `undefined`   |
```

File: docs/README.md [hunk 5]
```diff
@@ -111,26 +111,25 @@ For example:
...
+| `installmentsToShow` | `number[]` | Determine the installment options you want to show to the user based on the number of installments. For example, if `[1, 3]` is passed as a value for this prop, only the installments options with `NumberOfInstallments` equal to 1 and 3 will be rendered. If a value is not passed, all options will be rendered. | `undefined`   |
```
Comment (julia-rabello) [inline docs/README.md:119]:
```suggestion
| `installmentsToShow` | `number[]` | Determines the installment options you want to show to the user based on the number of installments. For example, if `[1, 3]` is passed as a value for this prop, only the installments options with `NumberOfInstallments` equal to 1 and 3 will be rendered. If a value is not passed, all options will be rendered. | `undefined`   |
```

File: docs/README.md [hunk 6]
```diff
@@ -111,26 +111,25 @@ For example:
...
+| -------------------------- | --------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------- |
+| `installmentsCriteria`     | `max-quantity` or `max-quantity-without-interest`               | When set to `max-quantity`, the block will render the installment plan with the most number of installments. When set to `max-quantity-without-interest`, the block will render the installment plan with the most number of installments and **no interest**. Note that if this prop is set to `max-quantity-without-interest` and no installment plan matches the 'without interest' criterion, the component will fall back to the default behavior. | `max-quantity` |
```
Comment (julia-rabello) [inline docs/README.md:125]:
```suggestion
| `installmentsCriteria`     | `max-quantity` or `max-quantity-without-interest`               | When set to `max-quantity`, the block renders the installment plan with the most number of installments. When set to `max-quantity-without-interest`, the block renders the installment plan with the most number of installments and **no interest**. Note that if this prop is set to `max-quantity-without-interest` and no installment plan matches the 'without interest' criterion, the component will fall back to the default behavior. | `max-quantity` |
```

**Review by julia-rabello — CHANGES_REQUESTED** [review]

**Review by julia-rabello — APPROVED** [review]

**Review by hellofanny — APPROVED** [review]

**Review by icazevedo — APPROVED** [review]
Approving so it is mergeable

**Comment by GeorgeLimaDev** [comment]
image
@vtex-apps/store-framework-devs please merge it as well since I am not authorized to do so. Thanks.

**Pull request merged**